src/requestLogger.js

```diff
@@ -42,6 +52,11 @@ class RequestLogger {
       return;
     }
 
+    if (this.staticData && typeof this.staticData === 'object') {
+      let baseObject = Object.assign({}, this.staticData);
+      messageAsObject = Object.assign(baseObject, messageAsObject);
```

Comment:
This can be done as single line `Object.assign({}, this.staticData, messageAsObject);`

Reply:
Fixed.